DEVELOPING.md

```diff
@@ -107,7 +107,7 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
 ## Eng docs
 
 - [Life of an AMP *](https://docs.google.com/document/d/1WdNj3qNFDmtI--c2PqyRYrPrxSg2a-93z5iX0SzoQS0/edit#)
-- [AMP Layout system](https://docs.google.com/document/d/1YjFk_B6r97CCaQJf2nXRVuBOuNi_3Fn87Zyf1U7Xoz4/edit)
+- [AMP Layout system](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-layout.md)
```

Comment:
looking at our other markdown files it looks like it should be relative to the files themselves.

see `docs/create_page.md` as an example